### ISSUE TITLE
ErrorException when clicking on email: Attempt to read property "link" on null

### ISSUE THREAD

**Issue by tiritibambix** [LinkStackOrg/linkstack-docker#94]
I got this error when clicking on my email link. Tried with 3 different browsers.

I'm no expert so I'm not sure what kind of info I should give.

```
            $newOrder++;

        }


        return response()->json([

            'status' => 'OK',

            'linkOrders' => $linkNewOrders,

        ]);

    }



    //Count the number of clicks and redirect to link

    public function clickNumber(request $request)

    {

        $linkId = $request->id;

        $link = Link::find($linkId);

        $link = $link->link;    #**was outlined in red**


        Link::where('id', $linkId)->increment('click_number', 1);


        $response = redirect()->away($link);

        $response->header('X-Robots-Tag', 'noindex, nofollow');


        return $response;

    }


    //Download Vcard

    public function vcard(request $request)

    {

        $linkId = $request->id;
```

Let me know if there are furthers infos I can give.

EDIT:
I tried with another email adress (gmail) and it seems to work.
The adress I'd like people to write me to is like this : contact@firstnamelastname.com
So maybe it is a problem regarding the domain ?

**Comment by JulianPrieber** [comment]
I'd need a stack trace for the error.
On the error page should be a share button somewhere.
It would be great if you could paste that link here.

Please keep in mind that if your config contains SMTP server credentials which would be shared with this link, including your SMTP password.

Should this be the case, please do not share this error. If you're using the built-in SMTP service, this shouldn't be an issue.

**Comment by tiritibambix** [comment]
Hello, thank you for your answer.

I was changing the email to test other domains, which were working. Then, I reverted back to the domain I first had an issue with to issue you the stack trace. Now it works.
I'm sure there never was a typo because I copy / pasted the address from the same document. So I'm clueless.

I just noticed that clicking the "email" button on android using Fennec (firefox fork) as a brwoser lands on a page saying ```redirecting to mailto:address@domain.com.```
